lib/v2/njust/eo.js

```diff
@@ -1,39 +1,48 @@
-const date = require('@/utils/date');
 const cheerio = require('cheerio');
-const { getContent } = require('@/routes/universities/njust/eo/util');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const { getContent } = require('./utils');
 
 const map = new Map([
-    ['16tz', { title: '南京理工大学电光16 -- 通知公告', id: '/_t217/tzgg/list.htm' }],
-    ['16dt', { title: '南京理工大学电光16 -- 主任寄语', id: '/_t217/zrjy/list.htm' }],
-    ['17tz', { title: '南京理工大学电光17 -- 年级通知', id: '/_t689/njtz/list.htm' }],
-    ['17dt', { title: '南京理工大学电光17 -- 每日动态', id: '/_t689/mrdt/list.htm' }],
-    ['18tz', { title: '南京理工大学电光18 -- 年级通知', id: '/_t900/njtz_10234/list.htm' }],
-    ['18dt', { title: '南京理工大学电光18 -- 主任寄语', id: '/_t900/zrjy_10251/list.htm' }],
-    ['19tz', { title: '南京理工大学电光19 -- 通知公告', id: '/_t1163/tzgg_11606/list.htm' }],
-    ['19dt', { title: '南京理工大学电光19 -- 每日动态', id: '/_t1163/mrdt_11608/list.htm' }],
+    ['16tz', { title: '南京理工大学电光16 -- 通知公告', id: '/_t217/tzgg' }],
+    ['16dt', { title: '南京理工大学电光16 -- 主任寄语', id: '/_t217/zrjy' }],
+    ['17tz', { title: '南京理工大学电光17 -- 年级通知', id: '/_t689/njtz' }],
+    ['17dt', { title: '南京理工大学电光17 -- 每日动态', id: '/_t689/mrdt' }],
+    ['18tz', { title: '南京理工大学电光18 -- 年级通知', id: '/_t900/njtz_10234' }],
+    ['18dt', { title: '南京理工大学电光18 -- 主任寄语', id: '/_t900/zrjy_10251' }],
+    ['19tz', { title: '南京理工大学电光19 -- 通知公告', id: '/_t1163/tzgg_11606' }],
+    ['19dt', { title: '南京理工大学电光19 -- 每日动态', id: '/_t1163/mrdt_11608' }],
```

Comment:
`/eoe/19/dt` does not work

Reply:
The original site of `/eo/19/dt` is temporarily down and will be available in the future.